cmd/deploy/deploy.go

```diff
@@ -379,6 +391,9 @@ func (dc *DeployCommand) RunDeploy(ctx context.Context, deployOptions *Options)
 	oktetoLog.SetStage("")
 
 	if err != nil {
+		if err.Error() == "interrupt signal received" {
```

Comment:
can we use if err is ??https://github.com/okteto/okteto/blob/58682ebd38516583db8fe373582a8bcd628efd24/pkg/errors/errors.go#L127